dists/fs-repo-migrations/Makefile

```diff
@@ -1,21 +1,22 @@
-repo=http://github.com/ipfs/fs-repo-migrations
-releases=../../releases/fs-repo-migrations
+package = github.com/ipfs/fs-repo-migrations
+repo = ${package}
+distname = fs-repo-migrations
+
+releases = ../../releases/${distname}
 
 all: dist
 
 dist: versions
-	../../build-go.sh fs-repo-migrations github.com/ipfs/fs-repo-migrations versions
+	../../build-go.sh ${distname} ${repo} ${package} versions
 	cp versions $(releases)/versions
 
 versions: filtered_versions
-	git ls-remote -t --refs $(repo) | egrep -o "refs/tags/v(.*)" | sed 's/refs\/tags\///' | sort > tag_versions
+	git ls-remote -t --refs https://$(repo) | egrep -o "refs/tags/v(.*)" | sed 's/refs\/tags\///' | sort > tag_versions
```

Comment:
We may also want to handle the `gx/vx.x.x` tags

Reply:
We also should export those common targets into a separate .mk file if possible

Reply:
We didn't rewrite the tags on go-ipfs, cluster, etc. (projects where we actually cut releases) so this shouldn't be an issue.

But yeah, we should have a common makefile.

Reply:
Done.